packages/beacon-node/src/util/multifork.ts

```diff
@@ -53,3 +53,25 @@ export function getStateTypeFromBytes(
   const slot = bytesToInt(bytes.subarray(SLOT_BYTES_POSITION_IN_STATE, SLOT_BYTES_POSITION_IN_STATE + SLOT_BYTE_COUNT));
   return config.getForkTypes(slot).BeaconState;
 }
+
+/**
+ * TODO: find correct position
+ * 8 + 32 = 40
+ * ```
+ * class BeaconState(Container):
+ *   genesis_time: uint64 [fixed - 8 bytes]
+ *   genesis_validators_root: Root [fixed - 32 bytes]
+ *   slot: Slot [fixed - 8 bytes]
+ *   ...
+ * ```
+ */
+const SLOT_BYTES_POSITION_IN_LIGHTCLIENTUPDATE = 40;
```

Comment:
TODO: calc and add corrected position

Reply:
I think its at offset 0?

should look like:
```
update = {
  beacon: {
    slot
    ...
  }
  ...
}
```
first field in `update` is `beacon`, first field in `beacon` is `slot`